io/net/bonding.py.data/bonding.yaml

```diff
@@ -18,3 +18,5 @@ peer_ip: "85.85.85.85"
 peer_interfaces: "eth3"
 bond_name: "bond13"
 user_name: "root"
+peer_bond_needed: False
+wait_time: "10"
```

Comment:
can it be used only where it is needed? The intention to use this is to wait for the Bond interface to come up in the peer. If this is set to any large value, then the entire script will take a long time to finish